### PR TITLE
✨ add loggin and loggout management in the client

### PR DESCRIPTION
**Pull request by Quentintnrl**

### CODE REVIEW

File: include/client/client_struct.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 typedef struct client_struct_s {
     // auth client
     char *username;
-    int uid;
+    char *uuid;
```
Comment (Peralban) [inline include/client/client_struct.h:17]: Why you didn't use uuid_t ?

File: src/client/parse_response.c [hunk 2]
```diff
@@ -44,18 +44,45 @@ const server_response_t server_responses[] = {
...
 void parse_response(client_struct_t *client_struct)
 {
     int code = 0;
+    int i = 0;
 
     if (client_struct->responses_args == NULL)
         return;
     code = atoi(client_struct->responses_args[0]);
-    for (int i = 0; i < 29; i++) {
+    for (; i < 29; i++) {
```
Comment (Peralban) [inline src/client/parse_response.c:75]: Why 29 ? Can you use something more explicit.

**Review by Peralban — COMMENTED** [review]
Some comment, I wait your response to Approve

File: include/client/client_struct.h (hunk 1 above)
Comment (Quentintnrl) [reply include/client/client_struct.h:17]: because the param that i send to the log function is a char *

File: src/client/parse_response.c (hunk 2 above)
Comment (Quentintnrl) [reply src/client/parse_response.c:75]: Because their is 29 log functions

**Review by landryarki — APPROVED** [review]

**Review by Peralban — APPROVED** [review]

**Pull request merged**